tests/integration/defs/ray_orchestrator/RL/test_rl_perf_reproduce.py

```diff
@@ -0,0 +1,59 @@
+import json
+import tempfile
+from pathlib import Path
+
+import pytest
+from defs.common import venv_check_call
+from defs.conftest import integration_path, llm_models_root
+from transformers import AutoTokenizer
+
+
+@pytest.mark.skip_less_device(4)
+@pytest.mark.parametrize(
+    "tp_size, num_instances", [(2, 2), (1, 4)], ids=["tp2_instances2", "tp1_instances4"]
+)
+def test_rl_perf_reproduce(llm_venv, tp_size, num_instances):
```

Comment:
How do we plan to capture potential perf regression if the current test is about benchmarking throughput if I read it correctly?